### PR TITLE
docs: updates & rewrites to actors in store page

### PR DESCRIPTION
**Pull request by TC-MO**
update outdated screenshots
replace bold with emphasis for clarity
rewrites for clarity and language
fix vale errors

### CODE REVIEW

File: sources/platform/actors/running/store.md [hunk 1]
```diff
@@ -3,129 +3,125 @@ title: Actors in store
...
 ### Rental Actors
 
-Rental Actors are like free Actors, but with a slight difference. When you use a rental Actor, you pay a fee to its developer. This empowers the developer to dedicate more time and effort to their Actors, thus ensuring they are of the **highest quality** and receive **ongoing maintenance**.
+Rental Actors are Actors for which you have to pay a recurring fee to the developer after your trial period ends. This empowers the developer to dedicate more time and effort to their Actors, thus ensuring they are of the _highest quality_ and receive _ongoing maintenance_.
 
-![Free and rental Actors in Apify Store](./images/store/free_vs_paid_actors.png)
+![Rental Actor example](./images/store/rental-actor-example.png)
 
-Each rental Actor has a **free trial,** where the trial's length is always shown for each rental Actor.
+Each rental Actor has a _free trial_ period. The lenght of the trail is displayed in each Actor's page.
```
Comment (TheoVasilis) [inline sources/platform/actors/running/store.md:35]:
```suggestion
Each rental Actor has a _free trial_ period. The length of the trial is displayed on each Actor's page.
```

File: sources/platform/actors/running/store.md [hunk 2]
```diff
@@ -3,129 +3,125 @@ title: Actors in store
...
+##### Can I run rental Actors via API or the Apify client?
 
 Yes, when you are renting an Actor, you can run it using either our [API](/api/v2), [JavaScript](/api/client/js) or [Python](/api/client/python) clients as you would do with private or free public Actors.
 
-**Do I pay platform costs for running rental Actors?**
+##### Do I pay platform costs for running rental Actors?
 
-Yes, you will pay normal [platform usage costs](https://apify.com/pricing/actors) on top of the monthly Actor rental fee. The platform costs work exactly the same way as for free public Actors or your private Actors. You should find estimates of the cost of usage in each individual rental Actor's README ([see an example](https://apify.com/drobnikj/crawler-google-places#how-much-will-it-cost)).
+Yes, you will pay normal [platform usage costs](https://apify.com/pricing/actors) on top of the per month Actor rental fee. The platform costs work exactly the same way as for free public Actors or your private Actors. You should find estimates of the cost of usage in each individual rental Actor's README ([see an example](https://apify.com/drobnikj/crawler-google-places#how-much-will-it-cost)).
```
Comment (TheoVasilis) [inline sources/platform/actors/running/store.md:50]:
```suggestion
Yes, you will pay normal [platform usage costs](https://apify.com/pricing/actors) on top of the monthly Actor rental fee. The platform costs work exactly the same way as for free public Actors or your private Actors. You should find estimates of the cost of usage in each individual rental Actor's README ([see an example](https://apify.com/drobnikj/crawler-google-places#how-much-will-it-cost)).
```

Comment (TheoVasilis) [reply sources/platform/actors/running/store.md:50]: Not sure why you changed that. It was fine (and better) as it was.

File: sources/platform/actors/running/store.md [hunk 3]
```diff
@@ -3,129 +3,125 @@ title: Actors in store
...
 
 You don't need a paid plan to start a rental Actor's free trial. Just activate the trial, and you are good to go. After that, you will need to subscribe to one of [Apify's paid plans](https://apify.com/pricing) in order to keep renting the Actor and continue using it.
 
-**When will I be charged for the Actor rental?**
+##### When will I be charged for the Actor rental?
 
 You always prepay the Actor rental for the following month. The first payment happens when the trial expires, and then recurs monthly. When you open the Actor in the Apify Console, you will see when the next rental payment is due, and you will also receive a notification when it happens.
 
-**Example**: You activate a 7-day trial of an Actor at **noon of April 1, 2021**. If you don't turn off auto-renewal, you will be charged at **noon on April 8, 2021**, then **May 8, 2021**, and so on.
+_Example_: You activate a 7-day trial of an Actor at _noon of April 1, 2021_. If you don't turn off auto renewal, you will be charged at _noon on April 8, 2021_, then _May 8, 2021_.
```
Comment (TheoVasilis) [inline sources/platform/actors/running/store.md:60]:
```suggestion
_Example_: You activate a 7-day trial of an Actor at _noon of April 1, 2021_. If you don't turn off auto-renewal, you will be charged at _noon on April 8, 2021_, then _May 8, 2021_.
```

File: sources/platform/actors/running/store.md [hunk 4]
```diff
@@ -3,129 +3,125 @@ title: Actors in store
...
 
-**Will I be automatically charged at the end of the free trial?**
+##### Will I be automatically charged at the end of the free trial?
 
-If you have an [Apify paid plan](https://apify.com/pricing), you will be automatically charged the monthly rental fee at the end of your free trial and you will be able to run the Actor for another month. If you are not subscribed to an Apify plan, you will need to subscribe to one of [Apify's paid plans](https://apify.com/pricing) in order to continue using the Actor after the trial has ended.
+If you have an [Apify paid plan](https://apify.com/pricing), the per month rental fee will be automatically subtracted from your plans' prepaid usage at the end of your free trial and you will be able to run the Actor for another month. If you are not subscribed to an Apify plan, you will need to subscribe to one of [Apify's paid plans](https://apify.com/pricing) in order to continue using the Actor after the trial has ended.
```
Comment (TheoVasilis) [inline sources/platform/actors/running/store.md:68]:
```suggestion
If you have an [Apify paid plan](https://apify.com/pricing), the monthly rental fee will be automatically subtracted from your plan's prepaid usage at the end of your free trial, and you will be able to run the Actor for another month. If you are not subscribed to any of [Apify's paid plans](https://apify.com/pricing), you will need to subscribe to one in order to continue using the Actor after the trial has ended.
```

File: sources/platform/actors/running/store.md [hunk 5]
```diff
@@ -3,129 +3,125 @@ title: Actors in store
...
 
-**Can I cancel my Actor rental?**
+##### Can I cancel my Actor rental?
 
-**You can cancel the Actor rental** during your trial or any time after that so that don't get charged when your current Actor rental expires. You can always turn it back on later if you want.
+_You can cancel the Actor rental_ during your trial or any time after that so that don't get charged when your current Actor rental period expires. You can always turn it back on later if you want.
```
Comment (TheoVasilis) [inline sources/platform/actors/running/store.md:72]:
```suggestion
_You can cancel the Actor rental_ during your trial or any time after that so you don't get charged when your current Actor rental period expires. You can always turn it back on later if you want.
```

File: sources/platform/actors/running/store.md [hunk 6]
```diff
@@ -3,129 +3,125 @@ title: Actors in store
...
 
-**Where can I see how much I have paid for Actor rental?**
+##### Where can I see how much I have paid for Actor rental?
 
 Since Actor rental fees are paid from prepaid platform usage, these fees conceptually belong under platform usage.
 
-You can find the breakdown of how much you have been charged for rental Actors at the bottom of the **Platform usage** tab in the [Billing](https://console.apify.com/billing) section. The overall sum will also be visible in the usage bar chart and the table at the top of the same tab.
+You can find the breakdown of how much you have been charged for rental Actors within **Actors** tab in **Current period** tab in the [Billing](https://console.apify.com/billing) section.
```
Comment (TheoVasilis) [inline sources/platform/actors/running/store.md:78]:
```suggestion
You can find the breakdown of how much you have been charged for rental Actors in the **Actors** tab, which you will find within the **Current period** tab in the [Billing](https://console.apify.com/billing) section.
```

File: sources/platform/actors/running/store.md [hunk 7]
```diff
@@ -142,10 +138,24 @@ at the bottom of the [Usage tab](https://console.apify.com/billing).
...
+
+_For more information on platform usage cost see the [usage and resources](./usage_and_resources.md) page._
+
+:::
+
 ## Reporting issues with Actors
 
-Each Actor has an `Issues` tab present in the Apify Console. There, you can open an issue (ticket) and chat with the Actor's author, platform admins,
-and other users of this Actor. Please, feel free to use the tab to ask any questions, request new features or give feedback. Alternatively, you can
+Each Actor has an **Issues** tab in the Apify Console. There, you can open an issue (ticket) and chat with the Actor's author, platform admins,
```
Comment (TheoVasilis) [inline sources/platform/actors/running/store.md:157]:
```suggestion
Each Actor has an **Issues** tab in Apify Console. There, you can open an issue (ticket) and chat with the Actor's author, platform admins,
```

File: sources/platform/actors/running/store.md [hunk 8]
```diff
@@ -142,10 +138,24 @@ at the bottom of the [Usage tab](https://console.apify.com/billing).
...
+and other users of this Actor. Please, feel free to use the tab to ask any questions, request new features or give feedback, or you can
```
Comment (TheoVasilis) [inline sources/platform/actors/running/store.md:158]:
```suggestion
and other users of this Actor. Please feel free to use the tab to ask any questions, request new features, or give feedback. Alternatively, you can
```

**Review by TheoVasilis — CHANGES_REQUESTED** [review]

File: sources/platform/actors/running/store.md [hunk 9]
```diff
@@ -3,129 +3,125 @@ title: Actors in store
...
+![Rental Actor example](./images/store/rental-actor-example.png)
 
-Each rental Actor has a **free trial,** where the trial's length is always shown for each rental Actor.
+Each rental Actor has a _free trial_ period. The lenght of the trail is displayed in each Actor's page.
 
-![Rental Actors in Apify Store](./images/store/paid-actors-store.png)
+![Rental Actors in Apify Store](./images/store/rental-actor-trial-example.png)
 
----
+After a trial period, a flat monthly _Actor rental_ fee is automatically subtracted from your prepaid platform usage in advance for the following month. This fee goes directly to the developer and is paid on top of the platform usage generated by the Actor. You can read more about our motivation for releasing rental Actors in [this blog post](https://blog.apify.com/make-regular-passive-income-developing-web-automation-actors-b0392278d085/) from Apify's CEO Jan Čurn.
```
Comment (mhamas) [inline sources/platform/actors/running/store.md:39]: `This fee goes directly to the developer`. This is actually no longer true, only 80% goes to developer. But maybe we don't have to be that specific. Let's just say `most of this fee goes directly to the developer`

File: sources/platform/actors/running/store.md [hunk 10]
```diff
@@ -3,129 +3,125 @@ title: Actors in store
...
+You can find the breakdown of how much you have been charged for rental Actors within **Actors** tab in **Current period** tab in the [Billing](https://console.apify.com/billing) section.
 
-![Rental Actors billing in Apify Console](./images/store/paid-actors-billing.png)
+![Rental Actors billing in Apify Console](./images/store/billing-paid-actors.png)
 
-### Actors paid per result
+### Pay per result
 
-When you run an Actor that is **paid per result**, you only pay for the successful results that an Actor returns when you run it, and you are not charged for the underlying platform usage. Technically, we validate the **results** against a schema, and you are only charged for the valid items in the [dataset](https://docs.apify.com/platform/storage/dataset) of the Actor run.
+When you run an Actor that is _paid per result_, you pay for the successful results that an Actor returns when you run it, and you are not charged for the underlying platform usage. Technically, we validate the _results_ against a schema, and you are only charged for the valid items in the [dataset](https://docs.apify.com/platform/storage/dataset) of the Actor run.
```
Comment (mhamas) [inline sources/platform/actors/running/store.md:84]: `Technically, we validate the _results_ against a schema, and you are only charged for the valid items in the [dataset](https://docs.apify.com/platform/storage/dataset) of the Actor run.`

Check this with Jarda or Zenda, I don't think everything is validated just yet.

File: sources/platform/actors/running/store.md [hunk 11]
```diff
@@ -3,129 +3,125 @@ title: Actors in store
...
 
-**Can I set a cap on how many results an Actor should return?**
 You can set a limit on how many items an Actor should return and the amount you will be charged in Options on the Actor detail page in the section below the Actor input.
 
 ![Max items for pay-per-result](./images/store/pay_per_result_max_items.png)
 
-**Can I publish an Actor that is paid per result?**
+##### Can I publish an Actor that is paid per result?
 
 Publishing an Actor that is paid per result is not yet available publicly as this pricing model is still in beta.
```
Comment (mhamas) [inline sources/platform/actors/running/store.md:122]: **Important:** It is now available publicly, anybody can do it.

**Review by mhamas — APPROVED** [review]

File: sources/platform/actors/running/store.md (hunk 2 above)
Comment (TC-MO) [reply sources/platform/actors/running/store.md:50]: it was to adhere to Msoft Style Guide, but after some consideration I think I will modify Vale setup to turn it off as it does not make much sense.

**Review by TheoVasilis — APPROVED** [review]

**Pull request merged**